packages/omg-js-util/src/transaction.js

```diff
@@ -113,9 +114,9 @@ const transaction = {
   *@returns the RLP encoded transaction
   *
   */
-  encode: function ({ transactionType, inputs, outputs, metadata, signatures }) {
+  encode: function ({ transactionType, inputs, outputs, metadata, signatures }, { signed = true } = {}) {
```

Comment:
Why is this `, { signed = true } = {})` instead of just `, signed = true`? Just curious. 😁

Reply:
then when you pass the option you know that it is `signed`
`encode(aaa, {signed: true})` instead of `encode(aaa, true)` the first one give the context of what is happening

Reply:
Gotcha 👍